### PR TITLE
Guid.xml updates for Try .NET

### PR DESCRIPTION
**Pull request by WilliamAntonRohm**
samples PR already merged -- https://github.com/dotnet/samples/pull/1654

### CODE REVIEW

File: xml/System/Guid.xml [hunk 1]
```diff
@@ -688,7 +690,7 @@
 ## Remarks  
  You can compare a GUID with the value of the <xref:System.Guid.Empty?displayProperty=nameWithType> field to determine whether a GUID is non-zero. The following example uses the <xref:System.Guid.op_Equality%2A> operator to compare two GUID values with <xref:System.Guid.Empty?displayProperty=nameWithType> to determine whether they consist exclusively of zeros.  
   
- [!code-csharp[System.Guid.Empty#1](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.guid.empty/cs/empty.cs#1)]
+ [!code-csharp-interactive[System.Guid.Empty#1](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.guid.empty/cs/empty.cs#1)]
```
Comment (mairaw) [inline xml/System/Guid.xml]: this is missing the note

File: xml/System/Guid.xml [hunk 2]
```diff
@@ -905,7 +907,7 @@
 ## Examples  
  The following code example creates and displays the values of two <xref:System.Guid> objects.
```
Comment (mairaw) [inline xml/System/Guid.xml]: this is missing the note

File: xml/System/Guid.xml [hunk 3]
```diff
@@ -960,7 +962,7 @@
 ## Examples  
  The following example uses the <xref:System.Guid.op_Equality%2A> operator to compare two GUID values with <xref:System.Guid.Empty?displayProperty=nameWithType> to determine whether they consist exclusively of zeros.
```
Comment (mairaw) [inline xml/System/Guid.xml]: this is missing the note

File: xml/System/Guid.xml [hunk 4]
```diff
@@ -1149,7 +1151,7 @@ After trimming, the length of the read-only character span is 0.
 ## Examples  
  The following example creates a new GUID, converts it to three separate string representations by calling the <xref:System.Guid.ToString%28System.String%29> method with the "B", "D", and "X" format specifiers, and then calls the <xref:System.Guid.Parse%2A> method to convert the strings back to <xref:System.Guid> values.  
   
- [!code-csharp[System.Guid.Parse#3](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.guid.parse/cs/parseex1.cs#3)]
+ [!code-csharp-interactive[System.Guid.Parse#3](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.guid.parse/cs/parseex1.cs#3)]
```
Comment (mairaw) [inline xml/System/Guid.xml]: this is missing the note. Add it to the MemberGroup element.

File: xml/System/Guid.xml [hunk 5]
```diff
@@ -1270,7 +1272,7 @@ The <xref:System.Guid.ParseExact%2A> method requires the read-only character spa
 ## Examples  
  The following example calls the <xref:System.Guid.ToString%28System.String%29> method with each of the supported format specifiers to generate an array of strings that represent a single GUID. These are then passed to the <xref:System.Guid.ParseExact%2A> method, which successfully parses only the string that conforms to the "B" format specifier.  
   
- [!code-csharp[System.Guid.Parse#4](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.guid.parse/cs/parseexactex1.cs#4)]
+ [!code-csharp-interactive[System.Guid.Parse#4](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.guid.parse/cs/parseexactex1.cs#4)]
```
Comment (mairaw) [inline xml/System/Guid.xml]: this is missing the note. Add it to the MemberGroup element.

File: xml/System/Guid.xml [hunk 6]
```diff
@@ -1450,7 +1452,7 @@ Because the `provider` parameter is ignored, you cannot use it to provide a cust
 ## Examples  
  The following example calls the <xref:System.Guid.NewGuid%2A> method to create a <xref:System.Guid> value, and then calls the <xref:System.Guid.ToByteArray%2A> method to represent the <xref:System.Guid> value as a byte array. It then displays both values to the console. Finally, it instantiates a new <xref:System.Guid> value from the byte array and calls its <xref:System.Guid.Equals%28System.Guid%29> method to show that the two <xref:System.Guid> values are identical.
```
Comment (mairaw) [inline xml/System/Guid.xml]: this is missing the note.

File: xml/System/Guid.xml [hunk 7]
```diff
@@ -1780,7 +1782,7 @@ Because the `provider` parameter is ignored, you cannot use it to provide a cust
 ## Examples  
  The following example creates a new GUID, converts it to three separate string representations by calling the <xref:System.Guid.ToString%28System.String%29> method with the "B", "D", and "X" format specifiers, and then calls the <xref:System.Guid.TryParse%2A> method to convert the strings back to <xref:System.Guid> values.  
   
- [!code-csharp[System.Guid.Parse#2](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.guid.parse/cs/tryparseex1.cs#2)]
+ [!code-csharp-interactive[System.Guid.Parse#2](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.guid.parse/cs/tryparseex1.cs#2)]
```
Comment (mairaw) [inline xml/System/Guid.xml]: this is missing the note

File: xml/System/Guid.xml [hunk 8]
```diff
@@ -1888,7 +1890,7 @@ Because the `provider` parameter is ignored, you cannot use it to provide a cust
 ## Examples  
  The following example calls the <xref:System.Guid.ToString%28System.String%29> method with each of the supported format specifiers to generate an array of strings that represent a single GUID. These are then passed to the <xref:System.Guid.TryParseExact%2A> method, which successfully parses the string that conforms to the "B" format specifier.  
   
- [!code-csharp[System.Guid.Parse#5](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.guid.parse/cs/tryparseexactex1.cs#5)]
+ [!code-csharp-interactive[System.Guid.Parse#5](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.guid.parse/cs/tryparseexactex1.cs#5)]
```
Comment (mairaw) [inline xml/System/Guid.xml]: this is missing the note

File: xml/System/Guid.xml [hunk 9]
```diff
@@ -241,8 +241,10 @@
   
 ## Examples  
  The following example passes each string listed in the Remarks section to the <xref:System.Guid.%23ctor%28System.String%29> constructor.  
-  
- [!code-csharp[System.Guid.ctor#1](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.guid.ctor/cs/ctor1.cs#1)]
+
+ [!INCLUDE[interactive-note](~/includes/csharp-interactive-note-some.md)]
```
[the remark this answers is not in the document]
Comment (WilliamAntonRohm) [reply xml/System/Guid.xml]: Done -- new lines 88-96

File: xml/System/Guid.xml (hunk 1 above)
Comment (WilliamAntonRohm) [reply xml/System/Guid.xml]: Added.

File: xml/System/Guid.xml (hunk 3 above)
Comment (WilliamAntonRohm) [reply xml/System/Guid.xml]: Added.

File: xml/System/Guid.xml (hunk 6 above)
Comment (WilliamAntonRohm) [reply xml/System/Guid.xml]: Added.

File: xml/System/Guid.xml (hunk 7 above)
Comment (WilliamAntonRohm) [reply xml/System/Guid.xml]: Added.

File: xml/System/Guid.xml (hunk 8 above)
Comment (WilliamAntonRohm) [reply xml/System/Guid.xml]: Added.

File: xml/System/Guid.xml (hunk 2 above)
Comment (WilliamAntonRohm) [reply xml/System/Guid.xml]: Added.

File: xml/System/Guid.xml (hunk 5 above)
Comment (WilliamAntonRohm) [reply xml/System/Guid.xml]: @mairaw -- Maira, I'm not sure where to add the interactivity note for this example (Parse#4). I've put an INCLUDE (new line 1288) just before this example -- the nearest previous MemberGroup is new line 709, for "Equals".

File: xml/System/Guid.xml (hunk 4 above)
Comment (WilliamAntonRohm) [reply xml/System/Guid.xml]: I've added an INCLUDE above this example -- please see my comment for old line 1273 below.

File: xml/System/Guid.xml (hunk 5 above)
Comment (WilliamAntonRohm) [reply xml/System/Guid.xml]: As per Parse#3, above.

File: xml/System/Guid.xml (hunk 4 above)
Comment (WilliamAntonRohm) [reply xml/System/Guid.xml]: I've updated this Member to a MemberGroup & moved the INCLUDE to the ##Remarks section.

**Comment by WilliamAntonRohm** [comment]
@mairaw -- Maira, this one is ready for your review.

**Comment by mairaw** [comment]
Need a new build

File: xml/System/Guid.xml [hunk 10]
```diff
@@ -85,6 +85,15 @@
       </AssemblyInfo>
       <Docs>
         <summary>Initializes a new instance of the <see cref="T:System.Guid" /> structure.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+  
+ [!INCLUDE[interactive-note](~/includes/csharp-interactive-note-some.md)]
```
Comment (mairaw) [inline xml/System/Guid.xml]:
```suggestion
 [!INCLUDE[interactive-note](~/includes/csharp-interactive-note.md)]
```

File: xml/System/Guid.xml [hunk 11]
```diff
@@ -241,8 +250,8 @@
   
 ## Examples  
  The following example passes each string listed in the Remarks section to the <xref:System.Guid.%23ctor%28System.String%29> constructor.  
-  
- [!code-csharp[System.Guid.ctor#1](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.guid.ctor/cs/ctor1.cs#1)]
+
```
Comment (mairaw) [inline xml/System/Guid.xml]:
```suggestion
  
```

File: xml/System/Guid.xml [hunk 12]
```diff
@@ -1081,7 +1096,7 @@ After trimming, the length of the read-only character span is 0.
         </exception>
       </Docs>
     </Member>
-    <Member MemberName="Parse">
+    <MemberGroup MemberName="Parse">
```
Comment (mairaw) [inline xml/System/Guid.xml]:
```suggestion
    <Member MemberName="Parse">
```

Comment (mairaw) [reply xml/System/Guid.xml]: You need to create a new entry for the MemberGroup like here:
https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System/Guid.xml#L698

File: xml/System/Guid.xml [hunk 13]
```diff
@@ -1161,7 +1176,7 @@ After trimming, the length of the read-only character span is 0.
         <altmember cref="M:System.Guid.TryParse(System.String,System.Guid@)" />
         <altmember cref="M:System.Guid.ToString" />
       </Docs>
-    </Member>
+    </MemberGroup>
```
Comment (mairaw) [inline xml/System/Guid.xml]:
```suggestion
    </Member>
```

File: xml/System/Guid.xml [hunk 14]
```diff
@@ -1212,7 +1227,7 @@ The <xref:System.Guid.ParseExact%2A> method requires the read-only character spa
         </remarks>
       </Docs>
     </Member>
-    <Member MemberName="ParseExact">
+    <MemberGroup MemberName="ParseExact">
```
Comment (mairaw) [inline xml/System/Guid.xml]:
```suggestion
    <Member MemberName="ParseExact">
```

same here. create a separate entry for membergroup

File: xml/System/Guid.xml [hunk 15]
```diff
@@ -1281,7 +1296,7 @@ The <xref:System.Guid.ParseExact%2A> method requires the read-only character spa
           <paramref name="input" /> is not in the format specified by <paramref name="format" />.</exception>
         <altmember cref="M:System.Guid.TryParseExact(System.String,System.String,System.Guid@)" />
       </Docs>
-    </Member>
+    </MemberGroup>
```
Comment (mairaw) [inline xml/System/Guid.xml]:
```suggestion
    </Member>
```

**Comment by WilliamAntonRohm** [comment]
@mairaw -- thank you for your detailed review.  I believe this PR is good to go!

File: xml/System/Guid.xml [hunk 16]
```diff
@@ -1144,12 +1159,12 @@ After trimming, the length of the read-only character span is 0.
   
  Use the <xref:System.Guid.TryParse%2A> method to catch any unsuccessful parse operations without having to handle an exception.  
   
-   
+ [!INCLUDE[interactive-note](~/includes/csharp-interactive-note.md)]
```
Comment (mairaw) [inline xml/System/Guid.xml]: move this note to a membergroup element

File: xml/System/Guid.xml [hunk 17]
```diff
@@ -1265,12 +1280,12 @@ The <xref:System.Guid.ParseExact%2A> method requires the read-only character spa
 |P|32 digits separated by hyphens, enclosed in parentheses:<br /><br /> (00000000-0000-0000-0000-000000000000)|  
 |X|Four hexadecimal values enclosed in braces, where the fourth value is a subset of eight hexadecimal values that is also enclosed in braces:<br /><br /> {0x00000000,0x0000,0x0000,{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00}}|  
   
+ [!INCLUDE[interactive-note](~/includes/csharp-interactive-note.md)]
```
Comment (mairaw) [inline xml/System/Guid.xml]: same here

File: xml/System/Guid.xml [hunk 18]
```diff
@@ -1780,7 +1797,9 @@ Because the `provider` parameter is ignored, you cannot use it to provide a cust
 ## Examples  
  The following example creates a new GUID, converts it to three separate string representations by calling the <xref:System.Guid.ToString%28System.String%29> method with the "B", "D", and "X" format specifiers, and then calls the <xref:System.Guid.TryParse%2A> method to convert the strings back to <xref:System.Guid> values.  
   
- [!code-csharp[System.Guid.Parse#2](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.guid.parse/cs/tryparseex1.cs#2)]
+ [!INCLUDE[interactive-note](~/includes/csharp-interactive-note.md)]
```
Comment (mairaw) [inline xml/System/Guid.xml]: same here

File: xml/System/Guid.xml (hunk 16 above)
Comment (WilliamAntonRohm) [reply xml/System/Guid.xml]: Done.

File: xml/System/Guid.xml (hunk 17 above)
Comment (WilliamAntonRohm) [reply xml/System/Guid.xml]: Done.

File: xml/System/Guid.xml (hunk 18 above)
Comment (WilliamAntonRohm) [reply xml/System/Guid.xml]: Done.

**Comment by WilliamAntonRohm** [comment]
@mairaw -- Maira, please review these changes if this build ever finishes :)

**Pull request merged**